<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThemeWidgetForm</class>
 <widget class="QWidget" name="ThemeWidgetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="themeLabel">
       <property name="text">
        <string>样本人数(M):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="sampleEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>100</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="animatedLabel">
       <property name="text">
        <string>初始财富值(N):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="wealthEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>100</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>每次交换财富值:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="exchangedEdit">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>努力的人(x):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="striveEdit">
       <property name="text">
        <string>20</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>税收:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="taxEdit">
       <property name="text">
        <string>0.2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>低保:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="protectEdit">
       <property name="text">
        <string>20</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="legendLabel">
       <property name="text">
        <string>模式:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="modeCombox">
       <item>
        <property name="text">
         <string>不负债</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>负债</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>税收和社保</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>x个努力的人</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startBtn">
       <property name="text">
        <string>开始模拟</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopBtn">
       <property name="text">
        <string>停止模拟</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetBtn">
       <property name="text">
        <string>重置</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>updateUI()</slot>
 </slots>
</ui>
